crates/deltalake-core/src/kernel/arrow/mod.rs

```diff
@@ -142,24 +150,7 @@ impl TryFrom<&DataType> for ArrowDataType {
                     .into(),
             )),
             DataType::Array(a) => Ok(ArrowDataType::List(Arc::new(a.as_ref().try_into()?))),
```

Comment:
This one should actually be 'UTC' timezone. I did not notice before the error of our totally not supporting writing UTC Tz issues was coming from this line :P

PrimitiveType::Timestamp => {
                        // Issue: https://github.com/delta-io/delta/issues/643
                        Ok(ArrowDataType::Timestamp(TimeUnit::Microsecond, None))
                    }

Reply:
yes, this is the current behavior. since we had so much fun with timestamps I did not want to break this as well. In any case something we can hopefully fix before `0.,17`.